lib/foreman_openscap/engine.rb

```diff
@@ -121,12 +121,14 @@ class Engine < ::Rails::Engine
                      :resource_type => 'SmartProxy'
         end
 
-        role "Compliance viewer", %i[view_arf_reports view_policies view_scap_contents view_tailoring_files view_openscap_proxies]
+        role "Compliance viewer", %i[view_arf_reports view_policies view_scap_contents view_tailoring_files view_openscap_proxies],
+             "TODO"
         role "Compliance manager", %i[view_arf_reports view_policies view_scap_contents
                                       destroy_arf_reports edit_policies edit_scap_contents assign_policies
                                       create_policies create_scap_contents destroy_policies destroy_scap_contents
                                       create_tailoring_files view_tailoring_files edit_tailoring_files destroy_tailoring_files
-                                      view_openscap_proxies]
+                                      view_openscap_proxies],
+             "TODO"
```

Comment:
This one allows full access to compliance features to non-admin users.

Reply:
So the role description could be something like this?
`Role granting all permissions to compliance features to non-admin users.`

Reply:
sounds good to me